sklearn/metrics/tests/test_classification.py

```diff
@@ -2180,6 +2180,16 @@ def test_brier_score_loss():
     assert_almost_equal(
         brier_score_loss(['foo'], [0.4], pos_label='foo'), 0.36)
 
+    # correctly infer pos_label
```

Comment:
Should this be tested in test_common.py?

Reply:
I guess not, because we only infer pos_label in this way in brier_score_loss.